views/routes.jsx

```diff
@@ -16,16 +17,24 @@ export default function getRoutes(store) {
     }
   };
 
+  const loadPackages = () => {
```

Comment:
This would cause the refresh problem since the server does not have fetch imported in the query function. This feature was added/fixed in #42